dffml/high_level.py

```diff
@@ -258,3 +258,60 @@ async def predict(
                 )
                 if update:
                     await sctx.update(record)
+
+
+async def load(source: BaseSource, *args: str) -> AsyncIterator[Record]:
+    """
+    Returns records from a source.
+
+    For each record given, call
+    :py:func:`update <dffml.source.source.BaseSourceContext.update>` on the
+    source. Effectively saving all the records to the source.
```

Comment:
This paragraph needs updating

Reply:
oh !

Reply:
So do i need to remove that line

Reply:
Since it was copied from `save` it's not an accurate description of what's happening here. You should update it to describe what is happening in `load`.